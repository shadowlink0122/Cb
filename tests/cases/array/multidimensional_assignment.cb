// 2次元配列の代入と参照のテストケース

int main() {
    // 2次元配列の宣言
    int[2][3] matrix;
    
    // 要素への代入
    matrix[0][0] = 10;
    matrix[0][1] = 20;
    matrix[1][2] = 99;
    
    // 要素の参照と出力
    println("matrix[0][0] =", matrix[0][0]);
    println("matrix[0][1] =", matrix[0][1]); 
    println("matrix[1][2] =", matrix[1][2]);
    
    return 0;
}
