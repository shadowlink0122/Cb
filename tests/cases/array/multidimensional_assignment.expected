matrix[0][0] = 10
matrix[0][1] = 20
matrix[1][2] = 99
